locales/en/plugin__odf-console.json

```
@@ -228,11 +228,24 @@
   "Application list": "Application list",
   "Last sync": "Last sync",
   "Activity status": "Activity status",
+  "Ready": "Ready",
+  "Not ready": "Not ready",
+  "Target cluster:": "Target cluster:",
+  "Last available: ": "Last available: ",
+  "{{actionType}} readiness:": "{{actionType}} readiness:",
+  "Data last synced on:": "Data last synced on:",
+  "KubeObject last synced on:": "KubeObject last synced on:",
```

as per previous discussion:
```suggestion
  "Kubernetes object last synced on:": "KubeObject last synced on:",
```